File: src/my_project/easy_problems/from101to150/final_value_after_operations.py
from typing import List 

class Solution:
    def finalValueAfterOperations(self, operations: List[str]) -> int:
        count: int = 0

        for item in operations:
            if item == "X++" or item == "++X":
                count += 1
            else:
                count -= 1

        return count 


solution = Solution()

print(solution.finalValueAfterOperations(operations = ["++X","++X","X++"]))

File: my_project_typescript/easy_problems_typescript/from1to50/finalValueAfterOperations.ts
function finalValueAfterOperations(operations: string[]): number {

    let count: number = 0

    for (let item of operations){

        if (item === "X++" || item === "++X" ){
            count += 1
        }
        else {
            count -= 1
        }
    }

    return count

};



console.log(finalValueAfterOperations(["++X","++X","X++"]))